internal/vm-console-proxy-bundle/bundle-test-incorrect.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: v1
```

Comment:
What about to add here a comment to say what it's missing (incorrect)? E.g. `Incorrect because ConfigMap is missing`.

Reply:
done